config.py

```diff
@@ -379,6 +379,9 @@ def create_transaction(db):
     # logging in via OIDC or a non-database internal auth provider.
     FEATURE_USERNAME_CONFIRMATION = True
 
+    # Feature Flag: If set to true, Quay will run using FIPS compliant hash functions.
+    FEATURE_FIPS = False
```

Comment:
Is the plan to be eventually fully FIPS compliant? It looks like the crypto ciphers are only covered by FIPS 140.